nautobot/virtualization/tests/test_filters.py

```diff
@@ -39,8 +39,8 @@ def setUpTestData(cls):
         )
 
         cls.clusters = [
```

Comment:
We should probably change the name of `test_group` and `test_type` in `ClusterTestCase` in this file to `test_cluster_group` and `test_cluster_type`